libraries/cmake/formula/openssl/CMakeLists.txt

```diff
@@ -3,6 +3,14 @@ project(thirdparty_openssl)
 set(OPENSSL_VERSION "1.1.1t")
 set(OPENSSL_ARCHIVE_SHA256 "8dee9b24bdb1dcbf0c3d1e9b02fb8f6bf22165e807f45adeb7c9677536859d3b")
 
+if(DEFINED PLATFORM_WINDOWS AND "${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "ARM64")
+    # 1.1.1t is broken for windows-arm64 (between 1.1.1s and 1.1.1t, from 43d8f88)
+    # Following commit fixes it, but will be available only in 1.1.1u:
+    # https://github.com/openssl/openssl/commit/eec34033552c810132736c772d37547f097a
+    set(OPENSSL_VERSION "1.1.1s")
+    set(OPENSSL_ARCHIVE_SHA256 "c5ac01e760ee6ff0dab61d6b2bbd30146724d063eb322180c6f18a6f74e4b6aa")
+endif()
```

Comment:
We can address this in a later PR, but I think we should download 1.1.1t and then apply a patch with the fix for the build that 1.1.1u has.
We can apply patches using ExternalProject in CMake: https://cmake.org/cmake/help/latest/module/ExternalProject.html (PATCH_COMMAND)

Reply:
I'll update this PR with this, no problem.